typhon/nonlte/spectra/lineshape.py

```diff
@@ -104,11 +104,12 @@ def DopplerWind(Temp, FreqGrid, Para, wind_v, shift_direction=False):
     wind = wind_v.reshape(wind_v.size, 1)
     FreqGrid = FreqGrid.reshape(1, FreqGrid.size)
     deltav = Para[0]*wind/c
-    if shift_direction is 'Red' or 'red' or 'RED':
+    if shift_direction.lower() == 'red':
         D_effect = (deltav)
-    elif shift_direction is 'Blue' or 'blue' or 'BLUE':
+    elif shift_direction.lower() == 'blue':
         D_effect = (-deltav)
-    else:print('Set Direction, red shift or blue shift')
+    else:
+        raise ValueError('Set Direction, red shift or blue shift')
```

Comment:
I would further change so it says "\'red\' shift or \'blue\' shift" to emphasize that only the first word matters.